ascent_macro/src/ascent_hir.rs

```diff
@@ -57,6 +57,7 @@ impl AscentConfig {
       ];
       for attr in attrs.iter() {
          if !recognized_attrs.iter().any(|recognized_attr| attr.meta.path().is_ident(recognized_attr)) {
+            let recognized_attrs: Vec<_> = recognized_attrs.iter().map(|attr| format!("`{attr}`")).collect();
```

Comment:
It is, since `.join()` is unfortunately only [implemented](https://doc.rust-lang.org/std/primitive.slice.html#method.join) on `slice` and thus unavailable for `T: Iterator`.

Reply:
There apparently was an RFC for `Iterator::join` in 2022, but it [got abandoned](https://github.com/rust-lang/rust/issues/75638).

Reply:
`itertools` has a `join` function, I think you can use that one.

Reply:
Fixed.